lang/cs/Org.Apache.REEF.IMRU/OnREEF/IMRUTasks/TaskCloseCoordinator.cs

```diff
@@ -56,6 +56,7 @@ private TaskCloseCoordinator()
         /// <param name="cancellationTokenSource"></param>
         internal void HandleEvent(ICloseEvent closeEvent, CancellationTokenSource cancellationTokenSource)
         {
+            Logger.Log(Level.Info, "HandleEvent: The task received close event");
```

Comment:
We don't have task id available at this stage, but this log is per task, so we know which task it is.